pkg/autoscaler/metrics/stat.proto

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package metrics;
+
+// Stat defines a single measurement at a point in time
+message Stat {
+    // The unique identity of this pod.  Used to count how many pods
+	// are contributing to the metrics.
+	string podName = 1;
```

Comment:
```suggestion
	string pod_name = 1;
```
This is the standard proto naming scheme.
The generator will generate `PodName` for go struct.